pyutilib/dev/runtests.py

```diff
@@ -131,6 +131,9 @@ def run(package, basedir, argv, use_exec=use_exec, env=None):
     else:
         cmd = ['nosetests']
 
+    if (platform == 'win' and sys.version_info[0:2] >= (3, 8)):
```

Comment:
Do you know if cygwin is affected, or only the native Windows builds?

Reply:
I'm not sure honestly. I tried it on my Windows VM and in the Github Actions windows runner.